.ado/jobs/cli-init.yml

```diff
@@ -296,14 +302,18 @@ jobs:
   - ${{ each config in parameters.buildMatrix }}:
     - ${{ if eq(config.BuildEnvironment, parameters.buildEnvironment) }}:
       - ${{ each matrix in config.Matrix }}:
-        - ${{ if eq(coalesce(matrix.useNuGet, false), parameters.buildNuGetOnly) }}:
+        - ${{ if and(eq(coalesce(matrix.useNuGet, false), parameters.buildNuGetOnly), eq(coalesce(matrix.useWinAppSDK, false), parameters.buildWinAppSDKOnly))}}:
```

Comment:
this line deserves a comment :)

Reply:
@dannyvv  do you think we're okay to remove the coalesce(<>, false) to simplify this a bit? since both useNuGet and useWinAppSDK default to false.

Reply:
Turns out when removed, PR checks fail right away with "Stage CLI must contain at least one job with no dependencies" so this will have to stay.